pkg/dialer/factory.go

```diff
@@ -136,7 +136,7 @@ func (f *Factory) clusterDialer(clusterName, address string) (dialer.Dialer, err
 
 	hostPort := hostPort(cluster)
 	logrus.Debugf("dialerFactory: apiEndpoint hostPort for cluster [%s] is [%s]", clusterName, hostPort)
-	if address == hostPort && isCloudDriver(cluster) {
+	if isCloudDriver(cluster) {
```

Comment:
@ibuildthecloud Do you have any security concerns
or any other concerns about hostPort/address check drop? Not sure what requests **do not go** to the apiEndpoint ip address in the hosted cluster, and you want them to go through the cluster agent